tds/routers/qualifierxref.py

```diff
@@ -0,0 +1,127 @@
+"""
+router.qualifier xrefs - crud operations for qualifier xrefs and
```

Comment:
I think leaving xref crud exposed so that other applications can manage them as orchestrators makes sense unless we want to start solely carrying some of that orchestration burden.

Reply:
TDS provides an abstraction above Postgres (+ Neo4j and ES). The API Schemas use nesting for example which hides the flat structure of the DB to the API user. XREFs are the way many-to-many relationships are handled specifically in SQL but we'd rather us write handle subobjects rather than basically let the orchestrator write to the DB using a very light wrapper